tests/test_finder.py

```diff
@@ -105,4 +105,15 @@ def test_setuptools_file_handling(tmp_path):
     """
     pth_file = tmp_path / "__editable__whatever.py"
     pth_file.write_text("hello\nMAPPING = {'griffe': 'src/griffe'}")
-    assert _handle_editable_module(pth_file) == Path("src")
+    assert list(_handle_editable_module(pth_file)) == [Path("src")]
```

Comment:
Same remark here about wrapping in `list`.

Reply:
Fixed